Adding entry for the slow solvers (#766)
conda/infrastructure: HOW_WE_USE_GITHUB.md

</details>

<details>
<summary><b>Slow solving of conda environment</b></summary>

<!-- this is a code block, any links need to be embedded -->
<pre>

> Hi user,

Reviewer: Just in case there's copypasta-ing, let's change this a bit
```suggestion
Hello!
```

> Hi <user>,

Author: ```suggestion
Hi user,
```

Thanks for voicing your concern about the performance of our dependency solver. To fix this, our official recommendation is using the "conda-libmamba-solver" instead of the default "classic" solver (more information about the "conda-libmamba-solver" can be found here: https://conda.github.io/conda-libmamba-solver/getting-started/).

> We know the old solver is slow which is why we put so much effort into to enabling the libmamba solver to be used within conda. We hope this provides you with a much better experience going forward.

Reviewer 2: ```suggestion
In most cases "conda-libmamba-solver" should be significantly faster than the "classic" solver. We hope it provides you with a much better experience going forward.
```
